Client/Frontend/Browser/BrowserViewController/BrowserViewController+Onboarding.swift

```diff
@@ -75,16 +75,22 @@ extension BrowserViewController {
   }
 
   func showNTPOnboarding() {
-    if Preferences.General.isNewRetentionUser.value == true,
-      Preferences.DebugFlag.skipNTPCallouts != true,
-      !topToolbar.inOverlayMode,
-      topToolbar.currentURL == nil,
```

Comment:
You still need the checks related with toolbar here

```
      !topToolbar.inOverlayMode,
      topToolbar.currentURL == nil,
```

Reply:
Also the debug flag will be useful `Preferences.DebugFlag.skipNTPCallouts` not to remove.